src/platform-includes/getting-started-verify/kotlin-multiplatform.mdx

```diff
@@ -0,0 +1,11 @@
+```kotlin
+import io.sentry.kotlin.multiplatform.Sentry
+
+fun captureError() {
+    try {
+      throw Exception("This is a test.")
+    } catch (e: Exception) {
```

Comment:
Missing import of the `Exception`

Reply:
Kotlin doesn't require to import Exception (double checked in IDE)